app/models/member.rb

```diff
@@ -23,6 +23,15 @@ class Member < ActiveRecord::Base
   validates_length_of :about_you, maximum: 255
 
   scope :subscribers, -> { joins(:subscriptions).order('created_at desc').uniq }
+  scope :not_banned, lambda {
```

Comment:
Use either `->` or `lambda` for consistency?

Reply:
I did initially but then codeclimate was complaining about using -> for multiline lambdas so I switched to the suggested format.